### PR TITLE
Local Search

### PR DESCRIPTION
**Pull request by Joshua-Jack**
Me and Louis got the local search working in the table, we had to merge in the code from figma0.5_nft to get the modal to not pop up.

### CODE REVIEW

File: src/components/css/nftpage.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    margin-bottom: 20px;
+  }
+  .info {
+    margin-bottom: 20px;
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+    .story {
+      height: 212px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:32]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  margin-top: 150px;
+  margin-left: $page-margin;
+  margin-right: $page-margin;
+  // border: 1px red solid;
+  display: flex;
+  flex-direction: column;
+
+  .showcase {
+    // remove
+    height: 542px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:16]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      width: 49%;
+      background: rgba(78, 0, 132, 0.2);
+      backdrop-filter: blur(30px);
+      border-radius: 16px;
+      border: 1px solid #bd5fff;
+      box-shadow: 0px 0px 10px 0px darken(rgba(235, 0, 255, 1), 20%);
+    }
+    .quad {
+      position: relative;
+      height: 212px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:42]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+      justify-content: space-between;
+      & > div {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        &:first-child {
+          margin-bottom: 20px;
+        }
+        & > div {
+          height: 96px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:55]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    }
+  }
+  .stats {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 20px;
+
+    .growth {
+      height: 303px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:73]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+      width: 49%;
+      background: rgba(78, 0, 132, 0.2);
+      backdrop-filter: blur(30px);
+      border-radius: 16px;
+      border: 1px solid #bd5fff;
+      box-shadow: 0px 0px 10px 0px darken(rgba(235, 0, 255, 1), 20%);
+    }
+    .market {
+      height: 303px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:82]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+      box-shadow: 0px 0px 10px 0px darken(rgba(235, 0, 255, 1), 20%);
+    }
+  }
+  .bottom {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 30px;
+    .chat {
+      height: 791px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:97]: Can we avoid using fixed pixel heights?

File: src/components/css/nftpage.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+      width: 49%;
+      background: rgba(78, 0, 132, 0.2);
+      backdrop-filter: blur(30px);
+      border-radius: 16px;
+      border: 1px solid #bd5fff;
+      box-shadow: 0px 0px 10px 0px darken(rgba(235, 0, 255, 1), 20%);
+    }
+    .history {
+      height: 791px;
```
Comment (benefacto) [inline src/components/css/nftpage.scss:106]: Can we avoid using fixed pixel heights?

**Review by benefacto — CHANGES_REQUESTED** [review]
It would be cool if we could avoid using fixed pixel heights as it will limit responsiveness. Let me know if that's possible.

**Comment by LPSCRYPT** [comment]
@benefacto I merged in a working branch of the nft page into this branch to get some of the search features working

much of that code for the nft branch has since been changed

we can merge in this code, then I'll pull and merge into my local branch and overwrite the old changes with my newer ones

**Comment by benefacto** [comment]
> 
> 
> It would be cool if we could avoid using fixed pixel heights as it will limit responsiveness. Let me know if that's realistic or if we should address it later.

This will be addressed in a future pull request.

**Review by benefacto — APPROVED** [review]

**Pull request merged**